docs/zed/api.md

```diff
@@ -45,7 +45,7 @@ POST /pool
 | ---- | ---- | -- | ----------- |
 | name | string | body | **Required.** Name of the pool. Must be unique to lake. |
 | layout.order | string | body | Order of value storage in pool. Possible values: desc, asc. Default: asc. |
-| layout.keys | array<string> | body | Primary key(s) of pool. Default: ts. |
+| layout.keys | `array<string>` | body | Primary key(s) of pool. Default: ts. |
```

Comment:
Feels odd to put just this type in backticks. How about escaping the angle brackets with a backslash or changing them to something else (like "array[string]")?